functions/Copy-DbaDbAssembly.ps1

```diff
@@ -189,24 +191,24 @@ function Copy-DbaDbAssembly {
                     }
                 }
 
-                if ($destServer.Databases[$dbName].Assemblies.Name -contains $currentAssembly.name) {
+                if ($destDb.Query("SELECT name FROM sys.assemblies WHERE name = '$assemblyName'").name) {
```

Comment:
Using SQL because SMO was not accurately reflecting the db state even when calling refresh().

Reply:
interesting!